.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+name: Receive Dispatch and Pull Docs
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [docs]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  pull-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v4
+
+    - name: Pull docs from rv-docs repository
```

Don't abbreviate here

```suggestion
    - name: Pull documentation
```